sources/vega/snapshot/v1/snapshot.proto

```diff
@@ -31,11 +31,14 @@ message Snapshot {
   bytes metadata = 5;
 }
 
+// NodeHash represents an exported node from the AVL tree and contains all the data necessary to recontruct and identical copy of the 
+// AVL tree after we've serialised it
 message NodeHash {
-  string full_key = 1; // namespace<separator>key
-  string namespace = 2;
   string key = 3;
   string hash = 4;
+  int32 height = 5;
```

Comment:
is height not int64?

Reply:
This isn't the block height, its the height of the node in the AVL tree. All of the fields here are related to the nodes in the AVL tree. There are lots of comments explaining all these fields on the core side.

Reply:
Thanks. might be worth having them here too for auto generation of docs.